ampersand-multifield-view.js

```diff
@@ -4,12 +4,17 @@ var find = require('lodash.find');
 var View = require('ampersand-view');
 
 var MultiFieldView = View.extend({
+  /* jshint maxlen: false */
   template: [
     '<div>',
       '<p class="label" data-hook="label"></p>',
+      '<div data-hook="multi-message-container" class="message message-below message-error">',
```

Comment:
Not within this plugin, but they're the same classes provided in other field views. [They](https://github.com/AmpersandJS/ampersand-input-view/blob/master/ampersand-input-view.js) aren't used in those views either.